package.json

```diff
@@ -239,6 +251,35 @@
         "view": "code-review.list",
         "contents": "No review comments found.\nCreate a comment by selecting the lines the comment is related to, then right click and choose:\n\"Code Review: Add Note\"\n[learn more](https://marketplace.visualstudio.com/items?itemName=d-koppenhagen.vscode-code-review)\n[Add a note](command:codeReview.addNote)"
       }
+    ],
+    "colors": [
+      {
+        "id": "codereview.priority.green",
+        "description": "Color for comments with priority level 1 in code-review extension.",
+        "defaults": {
+          "dark": "#008000",
+          "light": "#008000",
+          "highContrast": "#008000"
+        }
+      },
+      {
+        "id": "codereview.priority.yellow",
+        "description": "Color for comments with priority level 2 in code-review extension",
+        "defaults": {
+          "dark": "#ffff00",
+          "light": "#ffff00",
+          "highContrast": "#ffff00"
+        }
+      },
+      {
+        "id": "codereview.priority.red",
+        "description": "Color for comments with priority level 3 in code-review extension",
+        "defaults": {
+          "dark": "#ff0000",
+          "light": "#ff0000",
+          "highContrast": "#ff0000"
+        }
+      }
```

Comment:
These settings should also be listed in the README.md

Reply:
Sorry, I forgot this one.
Done.